tests/natural_language/WriteGoodLintBearTest.py

```diff
@@ -0,0 +1,144 @@
+from bears.natural_language.WriteGoodLintBear import WriteGoodLintBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """The 70s era was awesome for music lovers.
+""".splitlines(keepends=True)
+
+bad_file = """So the 70s era was awesome for music lovers.
+""".splitlines(keepends=True)
+
+passive_file = """This analysis was provided by coala-bears.
+""".splitlines(keepends=True)
+
+Wordy_file = """It was a project which was very complex in structure and very
+ambitious in nature.""".splitlines(keepends=True)
+
+thereIs_file = """There are so many tools to check this, no rigtht?
+""".splitlines(keepends=True)
+
+cliche_file = """A fine kettle of fish.""".splitlines(keepends=True)
+
+adverb_file = """It really is extremely hot day.""".splitlines(keepends=True)
+
+so_file = """So it was a bad day,indeed?""".splitlines(keepends=True)
+
+illu_file = """Many readers are not aware that the the second the is ignored
+.""".splitlines(keepends=True)
+
+weasel_file = """Some people say that weasel words are great.
+""".splitlines(keepends=True)
+
+WriteGoodLintBearTest = verify_local_bear(WriteGoodLintBear,
+                                          valid_files=(good_file,),
+                                          invalid_files=(bad_file,))
+
+WriteGoodLintBearWithPassiveTest = verify_local_bear(WriteGoodLintBear,
+                                                     valid_files=(good_file,
+                                                                  weasel_file,
+                                                                  bad_file,
+                                                                  Wordy_file,
+                                                                  thereIs_file,
+                                                                  cliche_file,
+                                                                  adverb_file,
+                                                                  so_file,),
+                                                     invalid_files=(
+                                                         passive_file,
+                                                         illu_file),
+                                                     settings={"passive": True})
+
+WriteGoodLintBearWithTooWordyTest = verify_local_bear(WriteGoodLintBear,
```

Comment:
I'd again disagree. `Class names should normally use the CapWords convention.`
https://www.python.org/dev/peps/pep-0008/#class-names
http://visualgit.readthedocs.io/en/latest/pages/naming_convention.html#classes

`mixedCase is allowed only in contexts where that's already the prevailing style`
http://stackoverflow.com/questions/159720/what-is-the-naming-convention-in-python-for-variable-and-function-names